chain/chain/src/store.rs

```diff
@@ -549,7 +549,10 @@ impl ChainStore {
         &self,
         id: &CryptoHash,
     ) -> Result<Vec<ExecutionOutcomeWithIdAndProof>, Error> {
-        Ok(self.store.get_ser(ColTransactionResult, id.as_ref())?.unwrap_or_else(|| vec![]))
+        Ok(self
+            .store
+            .get_ser(ColTransactionResult, id.as_ref())?
+            .unwrap_or_else(std::vec::Vec::new))
```

Comment:
```suggestion
            .unwrap_or_default())
```

How `std::vec::Vec::new` get in here? This is not something one ever writes by hand...

Reply:
@matklad Oh, sorry. I fixed the issue.

Reply:
@matklad I made a contribution to `clippy`, `.unwrap_or_else(std::vec::Vec::new))` should be replaced by `.unwrap_or_default())` by default

https://github.com/rust-lang/rust-clippy/pull/8163

Reply:
Awesome, thanks @pmnoxx !